### ISSUE TITLE
adding and changing year not intuitive

### ISSUE THREAD

**Issue by npfoss** [npfoss/courseroad2#160]
Just did some user testing on one person, but...

- he struggled to find how to add a class
- definitely was never going to figure out how to change his year
- also was confused by that^ menu's labels "First year, etc". might be better to make it "when was your freshman fall?" and have calendar years
- he likes all the drag and drop a lot
- did not figure out you can drag from the audit, but that's post MVP for sure

Not sure how much of this we want to address for MVP....

**Comment by georgiashay** [comment]
Changing the menu's label's is easy enough for MVP, but unless you have an idea for indicating changing your class year other than the weird fuzzy green highlight that discussion is probably post-MVP.

**Comment by npfoss** [comment]
I agree. For MVP let's just replace "Change Class Year" with "I am a..." because it avoids the confusion with the overloaded word "class" as well.

**Comment by npfoss** [comment]
This may help with the adding classes thing: https://github.com/npfoss/courseroad2/pull/177#discussion_r276846627
